src/mongo/Config.class.php

```diff
@@ -2026,6 +2064,48 @@ public function getTransactionLogDatabase($readPreference = \MongoClient::RP_PRI
     }
 
     /**
```

Comment:
Agreed about traits - I've been longing for them.  I'm glad "this PR's lifetime" won't stretch that far, however :)